frontend/src/app/store.js

```diff
@@ -26,7 +26,7 @@ export const initialState = {
     data: experiments.results
   },
   addon: {
-    hasAddon: !!window.navigator.testpilotAddon,
+    hasAddon: null,
```

Comment:
Yes, it's gross but it works. The only other way I can think of fixing the situation is to move the "has addon" pages to a different url tree than "doesn't have add-on" pages, which I think we need to do for other architectural reasons too. Then we can stop using the null hack.